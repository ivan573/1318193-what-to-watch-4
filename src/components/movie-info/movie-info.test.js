import React from "react";
import renderer from "react-test-renderer";
import MovieInfo from "./movie-info.jsx";

test(`MovieInfo matches snapshot`, () => {
  const tree = renderer
    .create(<MovieInfo
      movie={{title: `Aviator`, genre: `Drama`, year: 2004}}
    />).toJSON;

  expect(tree).toMatchSnapshot();
});
