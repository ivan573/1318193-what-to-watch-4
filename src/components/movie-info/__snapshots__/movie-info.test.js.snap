// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieInfo matches snapshot 1`] = `[Function]`;
